chef.gemspec

```diff
@@ -16,6 +16,7 @@ Gem::Specification.new do |s|
   s.required_ruby_version = ">= 2.5.0"
 
   s.add_dependency "chef-config", "= #{Chef::VERSION}"
+  s.add_dependency "chef-core", "~> 0.0"
 
```

Comment:
We should leave a note here that we need to add the train-core 2.0 dep as soon as its released